chocolatey/plugins/modules/win_chocolatey_source.ps1

```diff
@@ -17,11 +17,11 @@ $spec = @{
         allow_self_service   = @{ type = "bool" }
         bypass_proxy         = @{ type = "bool" }
         certificate          = @{ type = "str" }
-        certificate_password = @{ type = "str" }
+        certificate_password = @{ type = "str"; no_log = $true }
         priority             = @{ type = "int" }
         source               = @{ type = "str" }
         source_username      = @{ type = "str" }
-        source_password      = @{ type = "str" }
+        source_password      = @{ type = "str"; no_log = $true }
         update_password      = @{ type = "str"; default = "always"; choices = "always", "on_create" }
```

Comment:
If source_password has no_log set to true, should update_password also be set like this?

Reply:
`update_password` isn't actually a password, it's more of a preference state that tells the module in what cases should it actually change the password if it's already set

Reply:
Oh that's kind of obvious when looking closer at the default and the choices :joy: